### DISCUSSION TITLE
If I already have qBit running when I instantiate a Client, would I have 2 instances running at the same time?

### DISCUSSION THREAD

**Discussion by ikki4** [rmartin16/qbittorrent-api#258] [General]
If I already have qBit running, and I run the command `qbt_client = qbittorrentapi.Client(**conn_info)`, what exactly happens? Would that new 'instance' just serve for sending commands to the already running qBit, or would that make a parallel instance that would send duplicated seeding data to my trackers?

**Comment by rmartin16** [comment]
Assuming the connection information, i.e. `conn_info`, remains the same, this would simply create another session in the running instance of qBittorrent. This would be the same as opening multiple incognito browser tabs and navigating all of them to the qBittorrent WebUI; each tab would be its own session with qBittorrent....but it's all the same qBittorrent instance they are talking to.

These sessions are pretty lightweight....but I wouldn't recommend opening hundreds of them relatively quickly without calling `client.auth_log_out()` on ones that won't be used anymore.

**Comment by ikki4** [comment]
Thanks for the clarification.
